tuyaha/tuyaapi.py

```diff
@@ -77,7 +84,6 @@ def get_access_token(self):
     def check_access_token(self):
         if SESSION.username == "" or SESSION.password == "":
             raise TuyaAPIException("can not find username or password")
-            return
```

Comment:
Why did you remove this `return`? I see no sense.

Reply:
Because is unreachable after a "raise"

Reply:
Ah, me stupid)))). You're right.